geometry_msgs/Point target_position
geometry_msgs/Vector3 wall_normal
---
std_msgs/Bool success